src/main/rules/GCI108/GCI108.json

```diff
@@ -0,0 +1,17 @@
+{
+    "title": "Comparison between XGBoost and RandomForest",
```

Comment:
could change title to give information about the real good practice and note a global title with no good practice to follow.
Maybe something like "Use XGBoost instead RandomForest" ...